bin/swagger_to_elm_client.py

```diff
@@ -63,7 +63,7 @@ def main() -> None:
         swagger_to.elm_client.write_client_elm(typedefs=elm_typedefs, requests=elm_requests, fid=fid_textio)
 
     pkg_pth = outdir / 'elm-package.sample.json'
-    elm_package_json = swagger_to.elm_client.elm_package_json()
+    elm_package_json = swagger_to.elm_client.elm_package_json(requests=elm_requests)
```

Comment:
I'd split this step into multiple functions for future extensibility:

* `needs_query_strings(requests) -> bool`
* `elm_package_json(query_strings: bool) -> MutableMapping[str, Any]`

This would split the view (`elm_package_json`) from the logic (`needs_query_strings`). In the future, we can add either more logic or more view in orthogonal ways. For example, other parts of the model may imply the need for query strings. Another example would be that we need to import a different library.

Reply:
I see your point! Makes sense. Update coming up.